src/App.js

```diff
@@ -1,50 +1,56 @@
-import './App.css';
+import './style/App.css';
 import { useEffect, useState } from "react";
-import { ShowPokemon } from './showPokemon';
+import { ShowPokemon } from './components//showPokemon';
```

Comment:
```suggestion
import { ShowPokemon } from './components/showPokemon';
```

Reply:
I'd also name file starting with capital letter, so `ShowPokemon` here

Reply:
Also - Do the name `ShowPokemon` sounds right? It starts with verb, but it's component, so should be more a noun